src/shared-render.test.js

```diff
@@ -0,0 +1,93 @@
+const {h, Component} = require('preact');
+
+const {render, shallow} = require('./preact-render-spy');
+
+const shape = (func, funcMember) => {
```

Comment:
also, you could probably just pass `funcMember` and destructure `{[funcMember]: func}` from the require:

```js

const renderSpy = require('./preact-render-spy');
const sharedTests = method => {
  const { [method]: func } = renderSpy;
};